gns3server/schemas/iou.py

```diff
@@ -182,7 +186,11 @@
         "command_line": {
             "description": "Last command line used by GNS3 to start QEMU",
             "type": "string"
-        }
+        },
+        "application_id": {
+            "description": "Application ID for running IOU image",
+            "type": ["integer", "null"]
```

Comment:
If I'm correct the OBJECT is not use for creation but just when we return the info. In this case the application_id should have been generated